edge-util/src/Microsoft.Azure.Devices.Edge.Storage/MemoryUsageAwareInMemoryDbStore.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I would wrap the InMemoryDbStore instead of extending

Reply:
Sure, will change to a decorator pattern